sdkconfig.defaults

```diff
@@ -7,8 +7,6 @@ CONFIG_PARTITION_TABLE_CUSTOM=y
 CONFIG_COMPILER_ORPHAN_SECTIONS_WARNING=y
 CONFIG_HTTPD_MAX_URI_LEN=2048
 CONFIG_HTTPD_WS_SUPPORT=y
-CONFIG_SPIRAM=y
-CONFIG_SPIRAM_MODE_OCT=y
 CONFIG_ESP_DEFAULT_CPU_FREQ_MHZ_240=y
 CONFIG_ESP_WIFI_11KV_SUPPORT=y
 CONFIG_SPIFFS_OBJ_NAME_LEN=64
```

Comment:
How about this? Is this default? I still recommend just reverting to the previous to avoid red herring situations.